src/components/interactive/StackedButton.jsx

```diff
@@ -10,10 +10,17 @@ const StackedButton = ({ Icon, Link, label, disabled, isActive, hasCaret, ...res
   return (
     <Element
       {...rest}
-      className={cx('fdsStackedButton', 'rounded--all', 'fdsStackedButton--ghost', {
-        'fdsStackedButton--disabled': disabled,
-        'fdsStackedButton--active': isActive,
-      })}
+      className={cx(
+        'fdsStackedButton',
+        'rounded--all',
+        'fdsStackedButton--ghost',
+        'border-focus--notrsn',
+        'trsn-default',
```

Comment:
in this case, what is setting the `transition-proprety`?

Reply:
Right. So again it really depends on how we want to sort of taper a more streamlined version for people. We can definitely have classes that just set properties and delays, but this class is more the recognition that in most of our cases right now (the case being all the interactive components), we can easily get by with the same transition delay, easing function, and property.

Reply:
I gave this some more thought and realized that there's always an option for an element to override `transition-property`, so I think you approach in setting that property in `transition--default` makes sense.

Let's say we want to use a standard transition, but only want to apply it to a single CSS property:

```html
<!-- transitions all properties by default -->
<div class="transition--default myClass" />
```

```css
.myClass {
        /* override to limit transition to background color only */
	transition-property: background-color;
}
```